doc/05_acting/02_acc.md

```diff
@@ -0,0 +1,76 @@
+# ACC - Adaptive Cruise Control
+
+**Summary:** This page contains information about the ACC component implemented in acting
+
+---
+
+## Author
+
+Julian Graf
+
+## Date
+
+06.02.2023
+
+<!-- TOC -->
+* [ACC - Adaptive Cruise Control](#acc---adaptive-cruise-control)
+  * [Author](#author)
+  * [Date](#date)
+  * [What is the ACC](#what-is-the-acc)
+  * [When to use the ACC](#when-to-use-the-acc)
+  * [How to turn on the ACC](#how-to-turn-on-the-acc)
+  * [How to turn off the ACC](#how-to-turn-off-the-acc)
+  * [How to test the ACC](#how-to-test-the-acc)
+  * [Also see](#also-see)
+<!-- TOC -->
+
+## What is the ACC
+
+The [ACC](https://en.wikipedia.org/wiki/Adaptive_cruise_control) will publish the vehicle speed to maintain a safe distance to vehicle ahead.
+An optimal distance to keep is therefore calculated using $dist = - \frac{1}{2} \cdot a \cdot (1s)^2 + v \cdot 1s + v \cdot 1s$.
+This gives 1 second to react and 1s to break before hitting the car in front if it suddenly stops.
+The target speed is calculated using a [PID controller](https://en.wikipedia.org/wiki/PID_controller).
+
+To do this, the node needs the current distance to the vehicle in front in meters via the topic `acc_distance`.
```

Comment:
The published velocity will be really big. But this should not be a problem in reality because we won't be able to detect a car in front, if it is half a kilometer away